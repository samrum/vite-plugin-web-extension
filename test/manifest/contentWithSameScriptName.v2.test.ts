import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("contentWithSameScriptName");

runManifestV2Test("contentWithSameScriptName", () => ({
  content_scripts: [
    {
      js: [`${resourceDir}/content1/content.js`],
      matches: ["https://*/*", "http://*/*"],
    },
    {
      js: [`${resourceDir}/content2/content.js`],
      matches: ["https://*/*", "http://*/*"],
    },
  ],
}));
