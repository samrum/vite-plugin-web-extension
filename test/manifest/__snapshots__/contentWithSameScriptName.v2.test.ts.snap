// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`contentWithSameScriptName - Manifest V2 1`] = `
[
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 2,
  \\"content_scripts\\": [
    {
      \\"js\\": [
        \\"test/manifest/resources/contentWithSameScriptName/content1/content.js\\"
      ],
      \\"matches\\": [
        \\"https://*/*\\",
        \\"http://*/*\\"
      ]
    },
    {
      \\"js\\": [
        \\"test/manifest/resources/contentWithSameScriptName/content2/content.js\\"
      ],
      \\"matches\\": [
        \\"https://*/*\\",
        \\"http://*/*\\"
      ]
    }
  ]
}",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/contentWithSameScriptName/content1/content.js",
    "name": undefined,
    "source": "console.log(\\"content1\\");
",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/contentWithSameScriptName/content2/content.js",
    "name": undefined,
    "source": "console.log(\\"content2\\");
",
    "type": "asset",
  },
]
`;
